npm-packages/convex/src/server/impl/query_impl.ts

```diff
@@ -326,7 +326,7 @@ export class QueryImpl implements Query<GenericTableInfo> {
       return null;
     }
     if (first_two_array.length === 2) {
-      throw new Error("unique() query returned more than one result");
+      throw new Error(`unique() query returned more than one result. First pair of results: ${first_two_array[0]._id} and ${first_two_array[1]._id}`);
```

Comment:
```suggestion
      throw new Error(`unique() query returned more than one result: 
 [${first_two_array[0]._id}, ${first_two_array[1]._id}, ...]`);
```

Suggestion for a little bit more terse way of showing it.
This is a great idea!

What do you think?

Reply:
Thanks, looks good to me! I have [opened a PR](https://github.com/get-convex/convex-helpers/pull/495) for basically the same problem in convex-helpers. The `unique()` function in that library just prints `"not unique"`. imo, it's confusing that the error messages are not unique.
I didn't even realize that `unique()` exists in both convex-helpers and the regular convex library. I saw the two different error messages in my logs and was very confused how `unique()` was ever called from convex-helpers in my repo. I traced it down to me using `customMutation/customQuery` from `convex-helpers/server/customFunctions.ts`.